### PR TITLE
feat(actions): add a workflow to automatically add new Zebra issues to projects

### PR DESCRIPTION
**Pull request by mpguerra**
## Motivation

We might want to use github projects instead of zenhub. 

## Solution

We need to automatically add new issues to the relevant Github projects

## Review

@gustavovalverde 

### Reviewer Checklist

  - [x] Will the PR name make sense to users?
    - [ ] Does it need extra CHANGELOG info? (new features, breaking changes, large changes)
  - [x] Are the PR labels correct?
  - [x] Does the code do what the ticket and PR says?
    - [ ] Does it change concurrent code, unsafe code, or consensus rules?
  - [ ] How do you know it works? Does it have tests?

## Follow Up Work

I want to add more automation to automatically set various project fields e.g  priority based on the priority labels

### CODE REVIEW

File: .github/project-management.yml
Comment (arya2) [inline .github/project-management.yml]:
```suggestion
  # Automatically add issues and PRs to the "Engineering Backlog" Github project.
  add-issue-to-zf-backlog-project:
    name: Adds all new issues to the "ZF Engineering Backlog" Github project
    runs-on: ubuntu-latest
    steps:
      - uses: actions/add-to-project@v0.4.0
        with: 
          project-url: https://github.com/orgs/ZcashFoundation/projects/13
          github-token: ${{ secrets.ACTIONS_PM_GITHUB_PROJECTS }}
```

File: .github/project-management.yml
Comment (arya2) [inline .github/project-management.yml]:
```suggestion
  # Automatically add issues and PRs to the "Zebra Backlog" Github project.
  add-issue-to-zebra-backlog-project:
```

File: .github/project-management.yml
Comment (arya2) [inline .github/project-management.yml]:
```suggestion
```

**Review by arya2 — COMMENTED** [review]
Looks good to me!

I left a few suggestions for condensing the duplicate event trigger.

**Review by gustavovalverde — APPROVED** [review]
LGTM, lets get it merged and test it

**Comment by teor2345** [comment]
This PR is causing a bug in Mergify, I have submitted a ticket.

The last PR batch got half-merged (which is a bug, but didn't cause any CI issues this time). As far as I can tell, the "must have approval from someone other than the last pusher" branch protection rule isn't checked consistently by Mergify (another bug). It seems to be checked before merge, but not while queueing.

Can we leave it alone until they get a chance to look at it overnight?

**Comment by gustavovalverde** [comment]
@Mergifyio update

**Review by teor2345 — APPROVED** [review]
Mergify has explained that GitHub doesn't have an API for the failing branch protection rule. So we might just have to live with the merge bug.

(Or not approve our own commits!)

**Comment by teor2345** [comment]
@mergifyio refresh

**Pull request merged**
